src/styles/Header.css

```diff
@@ -1,21 +1,25 @@
 /*-------------------------------------------------------------------------------
  HEADER
  -------------------------------------------------------------------------------*/
-#header {
+header {
   position: fixed;
   top: 0px;
   left: 0px;
   justify-content: space-between;
   align-items: center;
```

Comment:
@bill-feng probably we don't need `justify-content: space-between;` and `align-items: center;`?

Reply:
hai, fixed!